lib/collection/widgets/second_pass_screen.dart

```diff
@@ -11,9 +12,162 @@ class SecondPassScreen extends StatelessWidget {
     return BlocBuilder<CollectionBloc, CollectionState>(
       builder: (context, state) {
         return Scaffold(
-            appBar: AppBar(title: const Text('Second Pass')),
-            body: Center(
-                child: Text('DEMO Respondent name: ' + state.respondentName.value)));
+          appBar: AppBar(
+            title: const Text('Second Pass'),
+            actions: [
+              IconButton(
+                  onPressed: () =>
+                      Navigator.pushNamed(context, PageRouter.secondPassHelp),
+                  icon: const Icon(Icons.help))
+            ],
+          ),
+          body: ListView.builder(
+              padding: const EdgeInsets.all(2.0),
+              itemCount: state.foodItems.length,
+              itemBuilder: (context, index) {
+                // TODO: refactor to more atomized widgets
+                return Card(
+                  child: Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: Column(
+                      children: [
+                        Text(state.foodItems[index].name.value +
+                            ' consumed in the ' +
+                            state.foodItems[index].timePeriod.value),
+                        // TODO: fix RenderFlex overflow
+                        DropdownButtonFormField(
+                          value: state.foodItems[index].source.value,
+                          decoration: InputDecoration(
+                            icon: const Icon(Icons.kitchen),
+                            labelText: 'Food source',
+                            helperText: 'Where does the food come from',
+                            errorText: state.foodItems[index].source.invalid
+                                ? 'Select the food source'
+                                : null,
+                          ),
+                          items: const [
+                            // TODO: research how to handle empty string as
+                            // the selected value other than a blank menu item
+                            DropdownMenuItem(child: Text(''), value: ''),
```

Comment:
consider refactoring theses lists out of this widget for readability purposes

Reply:
Yup, I've refactored the whole `FoodItem` `Card` widgets to a separate widget in a standalone file, and then moved these lists of `DropdownMenuItem`s into separate lists defined in the `build` method of those widgets, since I think that is how they do it in Flutter BLoC examples, e.g: 
<img width="515" alt="image" src="https://user-images.githubusercontent.com/45038919/143442802-29f0adcb-9b37-4fa3-b7b9-d5043e41524d.png">